### PR TITLE
chore(deps): Bump aws-cdk from 1.61.1 to 1.66.0

### PR DESCRIPTION
**Pull request by grbartel**
Updates tests which broke due to to changing of cloudformation format and extra
Arn Validation added to CDK.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by ddneilson — DISMISSED** [review]
Great! Thanks Grant

File: packages/aws-rfdk/lib/deadline/test/render-queue.test.ts [hunk 1]
```diff
@@ -1460,7 +1460,7 @@ describe('RenderQueue', () => {
                   'DNSName',
                 ],
               },
-              ':4433" --tls-ca "caarn"\n' +
+              ':4433" --tls-ca "arn:aws:secretsmanager:123456789012:secret:ca/arn"\n' +
```
Comment (jericht) [inline packages/aws-rfdk/lib/deadline/test/render-queue.test.ts]: Could we change this to use the `CA_ARN` constant defined above?

File: packages/aws-rfdk/lib/deadline/test/render-queue.test.ts [hunk 2]
```diff
@@ -1631,7 +1631,7 @@ describe('RenderQueue', () => {
                   'DNSName',
                 ],
               },
-              ':4433" --tls-ca \"caarn\" 2>&1\n' +
+              ':4433" --tls-ca \"arn:aws:secretsmanager:123456789012:secret:ca/arn\" 2>&1\n' +
```
Comment (jericht) [inline packages/aws-rfdk/lib/deadline/test/render-queue.test.ts]: Same thing here, using the `CA_ARN` constant

**Review by jericht — CHANGES_REQUESTED** [review]

File: packages/aws-rfdk/lib/deadline/test/render-queue.test.ts (hunk 1 above)
Comment (grbartel) [reply packages/aws-rfdk/lib/deadline/test/render-queue.test.ts]: This has been updated

**Review by jericht — APPROVED** [review]
Thanks Grant. LGTM.

**Review by ddneilson — APPROVED** [review]

**Pull request merged**